src/controllers/helpers/projects/projectHelpers.js

```diff
@@ -0,0 +1,169 @@
+import {
+  showTodoBody,
+  extractProject
+} from '../todos/todoBody';
+import projectsController from "../../projectsController";
+
+import {
+  generateAddTodoBtn
+} from "../todos/todoHelpers";
+
+export const showProjectList = () => {
+  // show project list only if there are projects in the projectsArray
+  let projectsArray = JSON.parse(localStorage.getItem("projectsArray"));
```

Comment:
This DOM helper should not have any idea of where the data is stored! (localStorage). Move this functionality (`localStorage.getItem("projectsArray")`) to its own file (probably `storage.js`)

Reply:
Solved